### ISSUE TITLE
Add postman templates to api documentation

### ISSUE THREAD

**Issue by sastels** [cds-snc/notification-planning#1268]
## Description

As a  client, I need to be able to learn how to use our api.

WHY are we building?
- client request. This would be an easy thing for us to do and possibly more users to figure out the api on their own.

WHAT are we building?
- Postman templates

VALUE created by our solution
- user experience
- increased self-service, reduce in support tickets 

## Acceptance Criteria
- [ ] templates created for a few cases of using the api

## Related
- [Helpdesk ticket](https://cds-snc.freshdesk.com/a/tickets/15266)

## QA Steps
- [ ] use the templates to make api calls

**Comment by yaelberger-commits** [comment]
cc @andrewleith if those postman files are helpful, maybe we can use them to create templates?

**Comment by andrewleith** [comment]
Yeah for sure wouldn't be too hard and could help new users quite a bit!

What do we want to create templates for? Here are some things we could provide templates for:
**Email**
- Sending an email without personalisation
- Sending an email with personalisation
- Sending a file as an attachment
- Sending a file as a link

**SMS**
- Sending an SMS without personalisation
- Sending an SMS with personalisation 

**BULK**
- Send a bulk email request
- Send a bulk SMS request

**Comment by yaelberger-commits** [comment]
Hey team! Please [add your planning poker estimate](https://app.zenhub.com/workspaces/Notify-Planning-614b3ad91bc2030015ed22f5/issues/gh/cds-snc/notification-planning/1268?planning-poker) with Zenhub @andrewleith @jzbahrai @smcmurtry @whabanks
